src/sqlittaaja/config.py

```diff
@@ -29,6 +29,8 @@ def validate_config(config: dict[str, Any]):
         raise MissingSectionError("answer")
     if config.get("exercise") is None:
         raise MissingSectionError("exercise")
+    if config.get("check_options") is None:
```

Comment:
I'm not sure if this is needed? What if user doesn't specify threshold; they would still need to specify this `check_options` section. You could default to this section being empty.

Reply:
Changed it to default empty and added value validation if it is privided.